### PR TITLE
Patrons : history tab

### PR DESCRIPTION
**Pull request by zannkukai**
## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/us/1283?milestone=260625

## How to test?

- Register some checkin for a patron (or register some checkin/checkout).
- Go into the new history tab and display loans informations.

Only CANCELLED and ITEM_RETURNED are listed into this tab ; Not ITEM_IN_TRANSIT_TO_HOUSE

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: projects/admin/src/app/circulation/patron/history/history-item/history-item.component.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              this.item = results[0];
+              this.document = results[1];
+            }
+          );
+        });
+    }
+  }
+
+  /**
+   * Get main title (corresponding to 'bf_Title' type, present only once in metadata)
```
Comment (AoNoOokami) [inline projects/admin/src/app/circulation/patron/history/history-item/history-item.component.ts:68]: Please add **@return** for each method.
Doc blocks must be written as this (empty line before and after method description)

```
/**
 * Description of the method
 *  @param describe parameters
 * @return describe what is returned
 */
```

File: projects/admin/src/app/pipe/provision-activity.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+@Pipe({
+  name: 'provisionActivity'
+})
+export class ProvisionActivityPipe implements PipeTransform {
+
+  /**
+   * extract the provision activity from the document metadata
+   * @param metadata: the document metadata (as `getRecord` with resolve=1)
+   * @param type: the type of provision to display. 'publication' by default
+   */
```
Comment (AoNoOokami) [inline projects/admin/src/app/pipe/provision-activity.pipe.ts:21]: Add **@return**

File: projects/admin/src/app/circulation/patron/history/history-item/history-item.component.ts (hunk 1 above)
Comment (zannkukai) [reply projects/admin/src/app/circulation/patron/history/history-item/history-item.component.ts:68]: Done.
Juste using the same method description used into other `getMainTitle` methods from other components

File: projects/admin/src/app/pipe/provision-activity.pipe.ts (hunk 2 above)
Comment (zannkukai) [reply projects/admin/src/app/pipe/provision-activity.pipe.ts:21]: done

**Pull request merged**